components/application-service/overlays/dev/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base/
+
+configMapGenerator:
+- literals:
+  - GITHUB_ORG=""
+  name: github-config
+  behavior: replace
```

Comment:
originally, this was used in `base` - which means for all environments (CPS & dev). The same is also used in pre-kcp version. 
TBH, I don't know the HAS implementation details, so only mentioning this so we don't miss anything.

Reply:
I moved this to dev since preview is changing GITHUB_ORG. And preview will be deploying only dev overlay. So I rather changed only in dev. I am not sure right now if the kcp-stable and kcp-unstable will be using same GITHUB_ORG if not then it can be easily patched in their overlay folders.